lib/bridge/IrcBridge.js

```diff
@@ -754,6 +769,128 @@ IrcBridge.prototype.getBotClient = function(server) {
     });
 }
 
+
+var _solveTimeout = null;
```

Comment:
What is this? I'm guessing a timeout ID? Or is it a time to wait? Or maybe a Promise? Please comment and/or give a clearer variable name. (quick grep at the code here looks like this is a [timeout ID](https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/clearTimeout))